src/openloco/windows/options.cpp

```diff
@@ -164,6 +160,8 @@ namespace openloco::ui::options
         make_remap_widget({ 158, 15 }, { 31, 27 }, widget_type::wt_8, 1, image_ids::tab, string_ids::miscellaneous_options)
 
         static constexpr int tabWidgets = (1 << widx::tab_display) | (1 << widx::tab_sound) | (1 << widx::tab_music) | (1 << widx::tab_regional) | (1 << widx::tab_controls) | (1 << widx::tab_miscellaneous);
```

Comment:
I like this idea I think we should make this another think we do on tabbed windows.